### PR TITLE
Sorting columns in admin statistic charts

### PR DESCRIPTION
**Pull request by ksierks**
## Description of changes
Sorted columns by date and changed daily format from "03/09" to "Mar 9" and monthly format from "07/21" to "Jan 2021". 
Note: Both daily and monthly formats start with month, which could be confusing when switching between time periods.
ex.) 03/21 could read as March 21 or March 2021.

image

image

## Related issue
Fixes issue #1066.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* ~[ ] Tests added (or description of how to test) for any new features.~
* ~[ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
This is looking great. Just a few changes. Let's remove the values that appear on top of the bars for either the Bar or Column chart types (if someone is viewing data for say the month then there could be 31 bars, and if a large number or projects, samples, or analyses were created on a certain day then the display could get messy. The values appear in the popover when we hover over a bar anyways so it's ok. I added a picture below in a comment of what it looks like with 31 bars.

Also, let's see if we can add some padding or something between the chart and the legend

**Comment by deepsidhu85** [comment]
adminstats

File: src/main/webapp/resources/js/pages/admin/chart-config.js [hunk 1]
```diff
@@ -41,15 +48,10 @@ export function getChartConfiguration(chartType, data) {
...
-    label: {
-      visible: Boolean(data),
-      position: "middle",
-      adjustColor: true,
-      style: { fill: "#0D0E68", fontSize: 12, fontWeight: 600, opacity: 0.3 },
-    },
     legend: {
       visible: Boolean(data),
       position: "bottom",
+      offsetY: 10
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/admin/chart-config.js:54]: Can I get you to add `maxWidth: window.innerWidth - 600` in the legend section as well. That way the legend doesn't extend from the chart margin on the left to the right

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
One last thing to add and it's good to go!

File: src/main/webapp/resources/js/pages/admin/chart-config.js (hunk 1 above)
Comment (ksierks) [reply src/main/webapp/resources/js/pages/admin/chart-config.js]: Please see 8c5ca22cc7d591a5d0a7bb21f6559dba0b638564.

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**